### PR TITLE
fix typo for nodelifecycle controller

### PR DESCRIPTION
**Pull request by mysunshine92**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
fix typo for nodelifecycle controller
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by mysunshine92** [comment]
ping @k82cn

**Comment by mysunshine92** [comment]
cc @bowei

File: pkg/controller/nodelifecycle/node_lifecycle_controller.go [hunk 1]
```diff
@@ -653,7 +653,7 @@ func (nc *Controller) doNoScheduleTaintingPass(ctx context.Context, nodeName str
 		return found
 	})
 	taintsToAdd, taintsToDel := taintutils.TaintSetDiff(taints, nodeTaints)
-	// If nothing to add not delete, return true directly.
+	// If nothing to add and delete, return true directly.
```
Comment (bowei) [inline pkg/controller/nodelifecycle/node_lifecycle_controller.go:656]: or

File: pkg/controller/nodelifecycle/scheduler/taint_manager.go [hunk 2]
```diff
@@ -103,7 +103,7 @@ func deletePodHandler(c clientset.Interface, emitEventFunc func(types.Namespaced
 	return func(ctx context.Context, args *WorkArgs) error {
 		ns := args.NamespacedName.Namespace
 		name := args.NamespacedName.Name
-		klog.V(0).InfoS("NoExecuteTaintManager is deleting pod", "pod", args.NamespacedName.String())
```
Comment (bowei) [inline pkg/controller/nodelifecycle/scheduler/taint_manager.go:106]: Is this a new recommended thing to do with logs? Why make this change?

**Review by bowei — COMMENTED** [review]
Not clear why we are making the logging change. You also don't mention it in your commit comment.

/hold

File: pkg/controller/nodelifecycle/scheduler/taint_manager.go (hunk 2 above)
Comment (mysunshine92) [reply pkg/controller/nodelifecycle/scheduler/taint_manager.go:106]: @bowei  klog.V(0).InfoS is equal to klog.InfoS, V(0) unnecessary

File: pkg/controller/nodelifecycle/node_lifecycle_controller.go (hunk 1 above)
Comment (mysunshine92) [reply pkg/controller/nodelifecycle/node_lifecycle_controller.go]: > or

@bowei i have modify

**Pull request merged**